ROADMAP.md

```diff
@@ -1,45 +1,40 @@
-    
-    - **Start generating code with Swift instead of Typescript**. This will allow much easier community contribution to code generation, and allow us to take on a bunch of improvements like `Hashable`, `Equatable`, and potentially `Identifiable` without having to fight with Typescript to do it.
-    - **Add immutable caching to new generated code**. Caching is currently heavily tied into our existing parsing mechanism. We're going to separate that out in two phases: The first will allow caching that *cannot* be changed by the consumer.
-    - **Add mutable caching to new generated code**. This is the final stage of updating caching: Allowing caching that *can* be changed by the consumer.
-    - **Remove Old Codegen**. Once all this is built, older codegen will be deprecated. 
-
-    You can follow this effort through the [Swift Codegen GitHub project](https://github.com/apollographql/apollo-ios/projects/2).
-    
-- **Cache Improvements**: We've had a ton of requests for this and we've got some extra muscle now, so we're going to be working on many requests. First up, some features inspired by Apollo Client 3 for the web: 
-    - **Better pagination support**. Better support for caching and updating paginated lists of objects. 
-    - **Reducing over-normalization**. Only separating out results into individual records when something that can identify them is present
-    - **Codable Support**: Which will allow easier moving between any storage method and objects, along with working better with updated codegen.
-
-- **General bug-bashing**: There are always a few outstanding general issues and small feature requests I'd like to address, and we'll be dealing with any new issues as they come up.
+- **Networking Stack Improvements**: The goal is to simplify and stabilise the networking stack.
+    - The [updated network stack](https://github.com/apollographql/apollo-ios/issues/1340) solved a number of long standing issues with the old barebones NetworkTransport but still has limitations. Switching to a composable request chain would enable building a pipeline out of reusable and customizable parts, giving users full control over the steps that are invoked to satisfy requests. This pattern has proven really useful for the JavaScript client (Apollo Link), and has been used succesfully in a number of other projects (like [OkHttp](https://square.github.io/okhttp/)).
 
+- **Improved Documentation and Tutorials**:
+    - Using [DocC](https://developer.apple.com/documentation/docc) would enable a richer, more integrated documentation experience.
 
-## Long-Term
+## 1.x/2.0 - _Future_
 
-These are things we plan to be working on **beyond the next 3 months**. 
+These are subject to change and anything that dramatically changes APIs or breaks backwards compatibility with 1.x releases will be reserved for the next major version.
```

Comment:
Let me know if there is anything else we want to specifically call attention to as something we'll be looking at in future versions.

Reply:
This looks like a good list from my perspective!